templates/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const HtmlPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+module.exports = {
+  entry: './src/scripts/index.js',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'build')
+  },
+  devServer: {
+    contentBase: './build',
+    port: 8000
+  },
+  devtool: isProduction ? false : 'source-map',
+  module: {
+    loaders: [
+      {
+        test: /\.jsx?$/,
```

Comment:
I'd say we should maybe have another project-template for react/preact. Also, what about preact-cli and create-react-app? They are customizable, aren't they?